### PR TITLE
Simplify instrument fields queries

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR simplifies the  instrument fields queries to make them async.

### CODE REVIEW

File: static/js/components/ObservationPlanRequestLists.jsx [hunk 1]
```diff
@@ -408,11 +408,27 @@ const ObservationPlanRequestLists = ({ gcnEvent }) => {
       const observationplanRequest =
         requestsGroupedByInstId[instrument_id][dataIndex];
 
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      const [isSubmitting, setIsSubmitting] = useState(false);
+      const handleSubmit = async () => {
```
Comment (stefanv) [inline static/js/components/ObservationPlanRequestLists.jsx:413]: Why async here?

**Review by stefanv — APPROVED** [review]

File: static/js/components/ObservationPlanRequestLists.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/ObservationPlanRequestLists.jsx]: Yup removing!

File: skyportal/handlers/api/instrument.py [hunk 2]
```diff
@@ -428,131 +404,51 @@ async def get(self, instrument_id=None):
...
-                        .unique()
-                        .all()
-                    )
-                data[ii]['fields'] = [
-                    {**tile.to_dict(), 'airmass': tile.airmass(time=airmass_time)}
-                    for tile, in tiles
-                ]
+                    data['fields'] = [
+                        {**tile.to_dict(), 'airmass': tile.airmass(time=airmass_time)}
+                        for tile, in tiles
```
Comment (guynir42) [inline skyportal/handlers/api/instrument.py:435]: can remove comma if you change `execute` to `scalars`

**Review by guynir42 — APPROVED** [review]
Looks good. Minor comments.

File: skyportal/handlers/api/instrument.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/instrument.py]: the sqlalchemy master.

**Pull request merged**